src/styles.scss

```diff
@@ -250,6 +250,7 @@ button,
   display: flex;
   align-items: center;
   justify-content: center;
+  user-select: text;
```

Comment:
This [supposedly](https://developer.mozilla.org/en-US/docs/Web/CSS/user-select#Browser_compatibility) allows to type into `<html>`, whatever that means (can't repro as I don't have access to Mac), but `user-select: auto` doesn't work.

Reply:
Is it necessary? On macOS Safari, I can still select the content of the textareas with `user-select: none` on the error splash.

Reply:
Not for the textareas, but the error message. Not really necessary, but expected behavior (if someone wants to google it or whatnot).